pulp_rpm/app/schema/modulemd.json

```diff
@@ -5,7 +5,10 @@
   "type": "object",
   "properties": {
     "name": {"type": "string"},
-    "version": {"type": "integer"},
+    "version": {
```

Comment:
Per the [spec](https://github.com/fedora-modularity/libmodulemd/blob/main/yaml_specs/modulemd_stream_v2.yaml#L65), it really ought to be an integer.  However it would not be the first time there was a bug in some tooling somewhere that produced something other than an integer.

What is the context, are you hitting an error?

Reply:
Yes, came across this in some Rocky 9 repos, for example:
```
curl -fsSL rockylinux.anexia.at/9.1/AppStream/x86_64/os/repodata/02fa9664-23dd-4215-bf82-c9c3bfb2c54d-MODULES.yaml.gz | gzip -d | grep version
```
Gives:
```
version: 1
version: 1
version: 1
version: 1
version: 2
    version: "9010020221119202858"
version: 2
    version: "9010020221119221509"
version: 2
    version: "9010020221206172252"
version: 2
    version: "9010020221119202900"
```
8.7 Seems to properly use integers:
```
curl -fsSL rockylinux.anexia.at/8.7/AppStream/x86_64/os/repodata/01fa634a16e45ab9aa2894048a90689edf293648226e7c0783dc926b16e796b9-modules.yaml.gz | gzip -d | grep version:| head
```
```
version: 2
  version: 8070020221108111110
version: 1
version: 2
  version: 8030020210429121623
version: 1
version: 2
  version: 8050020211109140251
version: 2
  version: 8050020220426164243
```

It's the sort of thing that, while not strictly correct, could help people avoid issues in the future. It should almost certainly be fixed upstream as well, but there isn't any downside in trying to catch and rectify this sort of thing down the line.

Reply:
> It should almost certainly be fixed upstream as well, but there isn't any downside in trying to catch and rectify this sort of thing down the line.

If possible we still try to avoid hacks like this, because it expands the potential for subtle bugs to sneak in over time.  Sometimes they're necessary though.

Could you file an issue with RockyLinux and link it here?

Reply:
https://bugs.rockylinux.org/view.php?id=2575

Reply:
Where are you thinking of adding it? JSON files don't support comments

Reply:
I suppose put it in the spot where the schema is checked in the code